### PR TITLE
Added blog for Efficient Filters in K-NN

### PR DESCRIPTION
**Pull request by navneet1v**
### Description
Added blog for Efficient Filters in K-NN
 
### Issues Resolved
https://github.com/opensearch-project/project-website/issues/1963

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-10-18-efficient-filters-in-knn.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title:  Efficient Filtering in OpenSearch Vector Engine
```
Comment (pajuric) [inline _posts/2023-10-18-efficient-filters-in-knn.md:3]: Please update the title to sentence case: 

Efficient filtering in OpenSearch vector engine

Comment (navneet1v) [reply _posts/2023-10-18-efficient-filters-in-knn.md:3]: sure.

File: _posts/2023-10-18-efficient-filters-in-knn.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title:  Efficient filtering in OpenSearch vector engine
+authors:
+- navneev
+- vamshin
+  date: 2023-10-18
+  categories:
+- technical-posts
+---
```
Comment (pajuric) [inline _posts/2023-10-18-efficient-filters-in-knn.md:10]: Please add the following meta: 

**meta_keywords:** efficient filtering in OpenSearch, OpenSearch vector engine, filtered vector similarity search, k-NN workloads in OpenSearch

**meta description:** Learn how using the inverted file (IVF) algorithm in OpenSearch improves performance of efficient filters and allows users to perform filtered vector similarity search at scale.

Comment (navneet1v) [reply _posts/2023-10-18-efficient-filters-in-knn.md:12]: updated and corrected

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - Blog is GTG and ready to publish.

**Review by nateynateynate — APPROVED** [review]
Looks good!

**Pull request merged**